pkg/kubectl/cmd/cmd_test.go

```diff
@@ -590,17 +557,11 @@ func Example_printPodShowAll() {
 
 func Example_printServiceWithNamespacesAndLabels() {
 	f, tf, _, ns := cmdtesting.NewAPIFactory()
-	p := printers.NewHumanReadablePrinter(nil, nil, printers.PrintOptions{
-		WithNamespace: true,
```

Comment:
Lost namespace option here

Reply:
> Lost namespace option here

The test faked a printer in a way the command couldn't run and then tested their fake printer. The test was dumb. It's either this is or removal. I don't care which.